autogen/agentchat/groupchat.py

```diff
@@ -44,6 +45,7 @@ class GroupChat:
     speaker_selection_method: Optional[str] = "auto"
     allow_repeat_speaker: Optional[Union[bool, List[Agent]]] = True
     enable_clear_history: Optional[bool] = False
+    send_introductions: Optional[bool] = False
```

Comment:
Perhaps using `bool` type hint? send_introductions doesn't seem to every been `None`?